lib/schema.ts

```diff
@@ -114,18 +121,20 @@ function buildFields(schema, rLevelParentMax, dLevelParentMax, path) {
               opts.fields,
               rLevelMax,
               dLevelMax,
-              path.concat([name]))
+              path.concat(name))
       };
-
+      
       if (opts.type == 'LIST' || opts.type == 'MAP') fieldList[name].originalType = opts.type;
 
```

Comment:
I was not sure if I was to remove this line or not. The `LIST` AND `MAP` types were commented out in types before so `opt.type` could never be one of the two according to typescript. I left it in anyways because a single test would fail otherwise